advanced/keyboard-shortcuts.md

```diff
@@ -11,47 +11,51 @@ keywords: keyboard shortcuts
+| Div                                          | Alt+Shift+8           | Ctrl+Option+8               | core                                                       |
+| Address                                      | Alt+Shift+9           | Ctrl+Option+9               | core                                                       |
+| Focus to menu bar                            | Alt+F9                | Option+F9                   | core                                                       |
+| Focus to toolbar                             | Alt+F10               | Option+F10                  | core                                                       |
+| Focus to element path                        | Alt+F11               | Option+F11                  | core                                                       |
+| Focus to contextual toolbar                  | Ctrl+F9               | Ctrl+F9                     | core                                                       |
+| Open the popup menu on split toolbar buttons | Shift+Enter           | Shift+Enter                 | core                                                       |
+| Open the help dialog                         | Alt+0                 | Option+0                    | [help]({{ site.baseurl }}/plugins/help/)                   |
+| Insert link                                  | Ctrl+K                | Command+K                   | [link]({{ site.baseurl }}/plugins/link/)                   |
+| Toggle Fullscreen                            | Ctrl+Shift+F          | Ctrl+Shift+F                | [fullscreen]({{ site.baseurl }}/plugins/fullscreen/)       |
+| Save                                         | Ctrl+S                | Command+S                   | [save]({{ site.baseurl }}/plugins/save/)                   |
+| Find                                         | Ctrl+F                | Command+F                   | [searchreplace]({{ site.baseurl }}/plugins/searchreplace/) |
 
 ## Accessibility keyboard shortcuts
 
 This is a list of available keyboard shortcuts within the editor user interface.
 
-| Action | Key |
-|--------|-----|
-| Execute command | Enter / Space |
-| Focus next widget | Right Arrow / Tab / Down Arrow |
-| Focus previous widget | Left Arrow / Shift+Tab / Up Arrow |
-| Open widget menu | Down Arrow |
-| Open submenu | Right Arrow |
-| Close submenu | Left Arrow |
-| Close dialog | Esc |
-| Close menu | Esc |
-| Move focus back to editor body | Esc |
+| Action                         | Key                               |
+|--------------------------------|-----------------------------------|
+| Execute command                | Enter / Space                     |
+| Focus on next UI Component     | Tab                               |
+| Focus on previous UI Component | Shift+Tab                         |
+| Focus next widget              | Right Arrow / Tab / Down Arrow    |
```

Comment:
I'm not sure widget is a good name here, maybe this should be `UI Component` as well?

Reply:
"widget" is a terrible name there. So I replaced it, also tested these shortcuts, they were wrong. Updated.